firestore/query.go

```diff
@@ -162,10 +163,19 @@ func (q Query) Offset(n int) Query {
 	return q
 }
 
-// Limit returns a new Query that specifies the maximum number of results to return.
-// It must not be negative.
+// Limit returns a new Query that specifies the maximum number of first results
+// to return. It must not be negative.
 func (q Query) Limit(n int) Query {
 	q.limit = &wrappers.Int32Value{Value: trunc32(n)}
+	q.limitToLast = nil
+	return q
+}
+
+// LimitToLast returns a new Query that specifies the maximum number of last
+// results to return. It must not be negative.
+func (q Query) LimitToLast(n int) Query {
+	q.limit = nil
+	q.limitToLast = &wrappers.Int32Value{Value: trunc32(n)}
```

Comment:
As I see it, `limitToLast` must drop `limit`, 'cause `limit` takes data from the start, while `limitToLast` takes from the end, so they are mutually exclusive